src/script/lua_api/l_server.cpp

```diff
@@ -171,14 +171,17 @@ int ModApiServer::l_get_player_information(lua_State *L)
 		return 1;                                                              \
 	}
 
-	ERET(getServer(L)->getClientConInfo(player->peer_id,con::MIN_RTT,&min_rtt))
-	ERET(getServer(L)->getClientConInfo(player->peer_id,con::MAX_RTT,&max_rtt))
-	ERET(getServer(L)->getClientConInfo(player->peer_id,con::AVG_RTT,&avg_rtt))
-	ERET(getServer(L)->getClientConInfo(player->peer_id,con::MIN_JITTER,&min_jitter))
-	ERET(getServer(L)->getClientConInfo(player->peer_id,con::MAX_JITTER,&max_jitter))
-	ERET(getServer(L)->getClientConInfo(player->peer_id,con::AVG_JITTER,&avg_jitter))
-
-	ERET(getServer(L)->getClientInfo(player->peer_id,
+	ERET(getServer(L)->getClientConInfo(player->getPeerId(), con::MIN_RTT, &min_rtt))
+	ERET(getServer(L)->getClientConInfo(player->getPeerId(), con::MAX_RTT, &max_rtt))
+	ERET(getServer(L)->getClientConInfo(player->getPeerId(), con::AVG_RTT, &avg_rtt))
+	ERET(getServer(L)->getClientConInfo(player->getPeerId(), con::MIN_JITTER,
+		&min_jitter))
+	ERET(getServer(L)->getClientConInfo(player->getPeerId(), con::MAX_JITTER,
+		&max_jitter))
+	ERET(getServer(L)->getClientConInfo(player->getPeerId(), con::AVG_JITTER,
+		&avg_jitter))
+
+	ERET(getServer(L)->getClientInfo(player->getPeerId(),
 										&state, &uptime, &ser_vers, &prot_vers,
```

Comment:
As you're changing/correcting this function, could you please also fix the indents here?